config/test.exs

```diff
@@ -33,6 +33,7 @@ config :logflare, Logflare.Repo,
 
 config :logflare, :postgres_backend_adapter, pool_size: 1
 config :grpc, start_server: false
+config :tesla, adapter: Tesla.Mock
```

Comment:
I have tried that in the past but there were a lot of issues. I am glad that we can land it right now. It will reduce amount of mocking that we need for HTTP clients.